webpack.dev.js

```diff
@@ -9,6 +9,7 @@ module.exports = merge(common, {
 		host: "0.0.0.0",
 		headers: {'Access-Control-Allow-Origin': '*'},
 		overlay: true,
+		disableHostCheck: true,
```

Comment:
why?

Reply:
exactly

Reply:
There's a ngrok plugin commented just below this, it's better to use that. Even better, run this:
```ssh -R 80:localhost:3000 serveo.net``` (change 3000 to whatever port it's running on your local machine).
Anyway, please remove this line when you finish, don't commit it, it can be a security risk.

Reply:
done